kubeflow/mpi-job/mpi-job.libsonnet

```diff
@@ -0,0 +1,58 @@
+local k = import "k.libsonnet";
+
+{
+  parts:: {
+    simple(namespace, name, gpus, image, command, args):: {
```

Comment:
The current thinking is that for the jobs the whole spec should be in the prototype file and not an import from a library. This way ks generate is the equivalent of copying an example. This makes it easier for users to do advanced modifications (e.g. set volume mounts).

Example:
https://github.com/kubeflow/kubeflow/blob/master/kubeflow/examples/prototypes/tf-job-simple.jsonnet